selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/SeleniumWebDriver.java

```diff
@@ -317,13 +319,22 @@ public void switchToNoneCurrentWindow(String currentWindowHandler) {
   }
 
   public void switchFromDashboardIframeToIde() {
-    new WebDriverWait(this, LOADER_TIMEOUT_SEC * 6)
-        .until(ExpectedConditions.frameToBeAvailableAndSwitchToIt(By.id("ide-application-iframe")));
+    switchFromDashboardIframeToIde(APPLICATION_START_TIMEOUT_SEC);
   }
 
   public void switchFromDashboardIframeToIde(int timeout) {
     new WebDriverWait(this, timeout)
-        .until(ExpectedConditions.frameToBeAvailableAndSwitchToIt(By.id("ide-application-iframe")));
+        .until(visibilityOfElementLocated(By.id("ide-application-iframe")));
```

Comment:
There is method `wait(timeout)` below in the class. You can use it to simplify the code.